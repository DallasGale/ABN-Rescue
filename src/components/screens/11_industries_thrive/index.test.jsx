/* eslint-env jest */

import React from 'react'
import { render } from '@testing-library/react'

import IndustriesThrive from '.'

describe('<IndustriesThrive />', () => {
  it('renders without crashing', () => {
    const component = render(<IndustriesThrive />)
    expect(component).toMatchSnapshot()
  })
})
