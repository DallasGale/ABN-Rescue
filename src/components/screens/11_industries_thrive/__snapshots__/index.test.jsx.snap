// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IndustriesThrive /> renders without crashing 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="styled__StyledWrapper-f9m6ts-0 dghsKP"
        style=""
      >
        <div
          class="scrollmagic-pin-spacer"
          data-scrollmagic-pin-spacer=""
          style="top: 0px; display: block; position: relative; box-sizing: content-box; min-width: 0px; width: auto; min-height: 0px; height: auto; padding-top: 0px; padding-bottom: 1000px;"
        >
          <div
            class="sticky"
            style="position: relative; margin: auto;"
          >
            <h2
              aria-hidden="true"
              class="typography__StyledMessageH2-sc-1n1kkhl-0 disappear styled__StyledMessage-f9m6ts-2 coaULs"
            >
              <div>
                <div
                  style="display: inline-block;"
                >
                  <div
                    style="display: inline-block; opacity: 0;"
                  >
                    A
                  </div>
                  <div
                    style="display: inline-block; opacity: 0;"
                  >
                    c
                  </div>
                  <div
                    style="display: inline-block; opacity: 0;"
                  >
                    r
                  </div>
                  <div
                    style="display: inline-block; opacity: 0;"
                  >
                    o
                  </div>
                  <div
                    style="display: inline-block; opacity: 0;"
                  >
                    s
                  </div>
                  <div
                    style="display: inline-block; opacity: 0;"
                  >
                    s
                  </div>
                   
                </div>
                <div
                  style="display: inline-block;"
                >
                  <div
                    style="display: inline-block; opacity: 0;"
                  >
                    t
                  </div>
                  <div
                    style="display: inline-block; opacity: 0;"
                  >
                    h
                  </div>
                  <div
                    style="display: inline-block; opacity: 0;"
                  >
                    e
                  </div>
                   
                </div>
                <div
                  style="display: inline-block;"
                >
                  <div
                    style="display: inline-block; opacity: 0;"
                  >
                    c
                  </div>
                  <div
                    style="display: inline-block; opacity: 0;"
                  >
                    o
                  </div>
                  <div
                    style="display: inline-block; opacity: 0;"
                  >
                    u
                  </div>
                  <div
                    style="display: inline-block; opacity: 0;"
                  >
                    n
                  </div>
                  <div
                    style="display: inline-block; opacity: 0;"
                  >
                    t
                  </div>
                  <div
                    style="display: inline-block; opacity: 0;"
                  >
                    r
                  </div>
                  <div
                    style="display: inline-block; opacity: 0;"
                  >
                    y
                  </div>
                </div>
                <br />
                <div
                  style="display: inline-block;"
                >
                  <div
                    style="display: inline-block; opacity: 0;"
                  >
                    t
                  </div>
                  <div
                    style="display: inline-block; opacity: 0;"
                  >
                    h
                  </div>
                  <div
                    style="display: inline-block; opacity: 0;"
                  >
                    e
                  </div>
                  <div
                    style="display: inline-block; opacity: 0;"
                  >
                    s
                  </div>
                  <div
                    style="display: inline-block; opacity: 0;"
                  >
                    e
                  </div>
                   
                </div>
                <div
                  style="display: inline-block;"
                >
                  <div
                    style="display: inline-block; opacity: 0;"
                  >
                    i
                  </div>
                  <div
                    style="display: inline-block; opacity: 0;"
                  >
                    n
                  </div>
                  <div
                    style="display: inline-block; opacity: 0;"
                  >
                    d
                  </div>
                  <div
                    style="display: inline-block; opacity: 0;"
                  >
                    u
                  </div>
                  <div
                    style="display: inline-block; opacity: 0;"
                  >
                    s
                  </div>
                  <div
                    style="display: inline-block; opacity: 0;"
                  >
                    t
                  </div>
                  <div
                    style="display: inline-block; opacity: 0;"
                  >
                    r
                  </div>
                  <div
                    style="display: inline-block; opacity: 0;"
                  >
                    i
                  </div>
                  <div
                    style="display: inline-block; opacity: 0;"
                  >
                    e
                  </div>
                  <div
                    style="display: inline-block; opacity: 0;"
                  >
                    s
                  </div>
                   
                </div>
                <div
                  style="display: inline-block;"
                >
                  <div
                    style="display: inline-block; opacity: 0;"
                  >
                    a
                  </div>
                  <div
                    style="display: inline-block; opacity: 0;"
                  >
                    r
                  </div>
                  <div
                    style="display: inline-block; opacity: 0;"
                  >
                    e
                  </div>
                   
                </div>
                <div
                  style="display: inline-block;"
                >
                  <div
                    style="display: inline-block; opacity: 0;"
                  >
                    t
                  </div>
                  <div
                    style="display: inline-block; opacity: 0;"
                  >
                    h
                  </div>
                  <div
                    style="display: inline-block; opacity: 0;"
                  >
                    r
                  </div>
                  <div
                    style="display: inline-block; opacity: 0;"
                  >
                    i
                  </div>
                  <div
                    style="display: inline-block; opacity: 0;"
                  >
                    v
                  </div>
                  <div
                    style="display: inline-block; opacity: 0;"
                  >
                    i
                  </div>
                  <div
                    style="display: inline-block; opacity: 0;"
                  >
                    n
                  </div>
                  <div
                    style="display: inline-block; opacity: 0;"
                  >
                    g
                  </div>
                  <div
                    style="display: inline-block; opacity: 0;"
                  >
                    .
                  </div>
                </div>
              </div>
            </h2>
            <div />
          </div>
        </div>
        <h2
          class="ally__screen-reader-only-text"
        >
          Across the country these industries are thriving.
        </h2>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="styled__StyledWrapper-f9m6ts-0 dghsKP"
      style=""
    >
      <div
        class="scrollmagic-pin-spacer"
        data-scrollmagic-pin-spacer=""
        style="top: 0px; display: block; position: relative; box-sizing: content-box; min-width: 0px; width: auto; min-height: 0px; height: auto; padding-top: 0px; padding-bottom: 1000px;"
      >
        <div
          class="sticky"
          style="position: relative; margin: auto;"
        >
          <h2
            aria-hidden="true"
            class="typography__StyledMessageH2-sc-1n1kkhl-0 disappear styled__StyledMessage-f9m6ts-2 coaULs"
          >
            <div>
              <div
                style="display: inline-block;"
              >
                <div
                  style="display: inline-block; opacity: 0;"
                >
                  A
                </div>
                <div
                  style="display: inline-block; opacity: 0;"
                >
                  c
                </div>
                <div
                  style="display: inline-block; opacity: 0;"
                >
                  r
                </div>
                <div
                  style="display: inline-block; opacity: 0;"
                >
                  o
                </div>
                <div
                  style="display: inline-block; opacity: 0;"
                >
                  s
                </div>
                <div
                  style="display: inline-block; opacity: 0;"
                >
                  s
                </div>
                 
              </div>
              <div
                style="display: inline-block;"
              >
                <div
                  style="display: inline-block; opacity: 0;"
                >
                  t
                </div>
                <div
                  style="display: inline-block; opacity: 0;"
                >
                  h
                </div>
                <div
                  style="display: inline-block; opacity: 0;"
                >
                  e
                </div>
                 
              </div>
              <div
                style="display: inline-block;"
              >
                <div
                  style="display: inline-block; opacity: 0;"
                >
                  c
                </div>
                <div
                  style="display: inline-block; opacity: 0;"
                >
                  o
                </div>
                <div
                  style="display: inline-block; opacity: 0;"
                >
                  u
                </div>
                <div
                  style="display: inline-block; opacity: 0;"
                >
                  n
                </div>
                <div
                  style="display: inline-block; opacity: 0;"
                >
                  t
                </div>
                <div
                  style="display: inline-block; opacity: 0;"
                >
                  r
                </div>
                <div
                  style="display: inline-block; opacity: 0;"
                >
                  y
                </div>
              </div>
              <br />
              <div
                style="display: inline-block;"
              >
                <div
                  style="display: inline-block; opacity: 0;"
                >
                  t
                </div>
                <div
                  style="display: inline-block; opacity: 0;"
                >
                  h
                </div>
                <div
                  style="display: inline-block; opacity: 0;"
                >
                  e
                </div>
                <div
                  style="display: inline-block; opacity: 0;"
                >
                  s
                </div>
                <div
                  style="display: inline-block; opacity: 0;"
                >
                  e
                </div>
                 
              </div>
              <div
                style="display: inline-block;"
              >
                <div
                  style="display: inline-block; opacity: 0;"
                >
                  i
                </div>
                <div
                  style="display: inline-block; opacity: 0;"
                >
                  n
                </div>
                <div
                  style="display: inline-block; opacity: 0;"
                >
                  d
                </div>
                <div
                  style="display: inline-block; opacity: 0;"
                >
                  u
                </div>
                <div
                  style="display: inline-block; opacity: 0;"
                >
                  s
                </div>
                <div
                  style="display: inline-block; opacity: 0;"
                >
                  t
                </div>
                <div
                  style="display: inline-block; opacity: 0;"
                >
                  r
                </div>
                <div
                  style="display: inline-block; opacity: 0;"
                >
                  i
                </div>
                <div
                  style="display: inline-block; opacity: 0;"
                >
                  e
                </div>
                <div
                  style="display: inline-block; opacity: 0;"
                >
                  s
                </div>
                 
              </div>
              <div
                style="display: inline-block;"
              >
                <div
                  style="display: inline-block; opacity: 0;"
                >
                  a
                </div>
                <div
                  style="display: inline-block; opacity: 0;"
                >
                  r
                </div>
                <div
                  style="display: inline-block; opacity: 0;"
                >
                  e
                </div>
                 
              </div>
              <div
                style="display: inline-block;"
              >
                <div
                  style="display: inline-block; opacity: 0;"
                >
                  t
                </div>
                <div
                  style="display: inline-block; opacity: 0;"
                >
                  h
                </div>
                <div
                  style="display: inline-block; opacity: 0;"
                >
                  r
                </div>
                <div
                  style="display: inline-block; opacity: 0;"
                >
                  i
                </div>
                <div
                  style="display: inline-block; opacity: 0;"
                >
                  v
                </div>
                <div
                  style="display: inline-block; opacity: 0;"
                >
                  i
                </div>
                <div
                  style="display: inline-block; opacity: 0;"
                >
                  n
                </div>
                <div
                  style="display: inline-block; opacity: 0;"
                >
                  g
                </div>
                <div
                  style="display: inline-block; opacity: 0;"
                >
                  .
                </div>
              </div>
            </div>
          </h2>
          <div />
        </div>
      </div>
      <h2
        class="ally__screen-reader-only-text"
      >
        Across the country these industries are thriving.
      </h2>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
